test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -314,6 +314,8 @@ def _run_dependency_check_test(image, ec2_connection):
         "CVE-2016-2182",
         # CVE-2020-13936: vulnerability found in apache velocity package which is a dependency for dependency-check package. Hence, ignoring.
         "CVE-2020-13936",
+        # CVE-2021-42550: affected file is part of dependency-check lib at path "dependency-check/lib/logback-core-1.2.3.jar" so we can ignore.
+        "CVE-2021-42550",
```

Comment:
So the disadvantage in this is that we ignore the CVE for any package - for instance, if it comes up on a package that's not dependency-check, we won't have a good mechanism to determine whether this is the case.

Working on testing the --exclude option for dependency check in #1611 - we can put this in temporarily to unblock builds and I'll remove it when merging in 1611